GraphWebhooks/Helpers/SubscriptionStore.cs

```diff
@@ -0,0 +1,42 @@
+﻿/*
+ *  Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+ *  See LICENSE in the source repository root for complete license information.
+ */
+
+using System;
+using System.Web;
+
+namespace GraphWebhooks.Helpers
+{
+    public class SubscriptionStore
+    {
+        public string SubscriptionId { get; set; }
+        public string ClientState { get; set; }
+        public string UserId { get; set; }
+        public string TenantId { get; set; }
+
+        private SubscriptionStore(string subscriptionId, Tuple<string, string, string> parameters)
+        {
+            SubscriptionId = subscriptionId;
+            ClientState = parameters.Item1;
+            UserId = parameters.Item2;
+            TenantId = parameters.Item3;
+        }
+
+        // This sample temporarily stores the current subscription ID, client state, user object ID, and tenant ID. 
+        // This info is required so the NotificationController can retrieve an access token from the cache and validate the subscription.
+        // Production apps typically use some method of persistent storage.
+        public static void SaveSubscriptionInfo(string subscriptionId, string clientState, string userId, string tenantId)
```

Comment:
We're not actually saving the refresh token anywhere?  If we're relying on the ADAL cache, is it persistent? 
One way to test this would be to:
1. have an implementation of this class that uses the file system (instead of HttpRuntime.Cache).  That makes it persistent. 
2. restart the process after subscribing to the notification.  When the webhook fires and we get invoked, I'm predicting that we'll fail to get a refresh token. 

We need to either a) push the refresh token into ADAL's cache;  b) configure ADAL's cache to be persistent.